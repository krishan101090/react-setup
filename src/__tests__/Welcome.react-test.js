import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Welcome from '../Welcome';

describe('Welcome (Snapshot)', () => {
  it('Welcome renders hello world', () => {
    const component = renderer.create(<Welcome />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});