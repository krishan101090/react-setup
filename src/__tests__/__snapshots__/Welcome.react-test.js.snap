// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome (Snapshot) Welcome renders  1`] = `
<div>
  Hi Hello React
</div>
`;

exports[`Welcome (Snapshot) Welcome renders hello world 1`] = `
<div>
  Hi Hello React
</div>
`;
